Projects/Book_store/bookStore/bookForm/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+# Create your models here.
+
+class Users(models.Model):
+    username = models.CharField(max_length = 40)
```

Comment:
@codal-uchoudhary , shouldn't we include password as well in here ? apart from username.

Reply:
Check this --> https://simpleisbetterthancomplex.com/tutorial/2016/07/22/how-to-extend-django-user-model.html

in the above link, read from the following section

Extending User Model Using a Custom Model Extending AbstractUser

Reply:
Actually , this is not for authentication purpose. for practice , i just submit the input data in database via form.

Reply:
and again fetch data from database and display the data .